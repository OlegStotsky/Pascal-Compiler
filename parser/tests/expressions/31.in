var 
a : integer;
b : integer;

procedure plus (first : float, second : float)
begin
	a := first + second;
end;

begin
	plus(a, b);
end;