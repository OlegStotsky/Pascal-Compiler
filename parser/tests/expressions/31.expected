Error at line 7, column 22 : can't cast type float to type integer
